Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -143,13 +132,15 @@ CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedStrin
     uint32_t length = 0;
     RETURN_NULL_IF_FAILED(localizedString->GetStringLength(index, &length));
 
-    // Get the string.
-    std::vector<wchar_t> wcharString = std::vector<wchar_t>(length + 1, 0);
-    RETURN_NULL_IF_FAILED(localizedString->GetString(index, wcharString.data(), length + 1));
+    // Get the string. Use length + 1 here as GetString requires room for the null terminator.
+    woc::unique_iw<wchar_t> wcharString(static_cast<wchar_t*>(IwMalloc(sizeof(wchar_t) * (length + 1))));
```

Comment:
unit tests still pass, and we have a number of tests that would exercise this, through CT/CGFont

---
In reply to: [92845292](https://github.com/Microsoft/WinObjC/pull/1558#discussion_r92845292) [](ancestors = 92845292)